Tests/Routing/ChainRouterTest.php

```diff
@@ -680,3 +710,7 @@ protected function createRouterMocks()
 abstract class RequestMatcher implements \Symfony\Component\Routing\RouterInterface, \Symfony\Component\Routing\Matcher\RequestMatcherInterface
 {
 }
+
+abstract class VersatileRouter implements \Symfony\Component\Routing\RouterInterface, \Symfony\Cmf\Component\Routing\ChainedRouterInterface, \Symfony\Component\Routing\Matcher\RequestMatcherInterface
```

Comment:
VersatileRouter already existed in master 
https://github.com/symfony-cmf/Routing/blob/master/Tests/Routing/ChainRouterTest.php#L741
I specifically had to add this to get the test working again